arch/arm/src/armv7-a/arm_dataabort.c

```diff
@@ -148,11 +148,14 @@ uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
 
 uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
 {
+  struct tcb_s *tcb = this_task();
+
   /* Save the saved processor context in current_regs where it can be
```

Comment:
Update the comment (current_regs is wrong). See if current_regs is mentioned elsewhere too?

Reply:
ok